modesty/pdf2json
# fix: issue #373: handle bad encoding expcetion by start page renderin…

…g after page operator list is resolved, add i373_err_broken.pdf to verify

to verify:
1. `npm run test` and `npm run test:forms` should all pass
2. `npm run test:misc` should see list of success and error and total counter matches expected
3. `npm run test:e` will parse i373_err_broken.pdf only, exception should be handled gracefully
